<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="VIBHJAIN-G2C5W" ignored="0" name="Test" tests="4" failures="1" timestamp="24 Feb 2018 08:02:23 GMT" time="0.08" errors="0">
  <testcase name="TestGrouping" time="0.009" classname="com.cisco.automation.modules.inv.TestInv">
    <failure type="java.lang.AssertionError" message="expected [10] but found [11]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [11]
	at com.cisco.automation.modules.inv.TestInv.TestGrouping(TestInv.java:65)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- TestGrouping -->
  <testcase name="TestHardAssertion" time="0.002" classname="com.cisco.automation.modules.inv.TestInv"/>
  <testcase name="TestSoftAssertion" time="0.005" classname="com.cisco.automation.modules.inv.TestInv"/>
  <testcase name="Test1" time="0.004" classname="com.cisco.automation.modules.fault.TestFault"/>
</testsuite> <!-- Test -->
